### PR TITLE
Add fdupes package

### PR DESCRIPTION
**Pull request by biyiklioglu**
Add fdupes 1.51,  a program for identifying or deleting duplicate files residing within specified directories.

### CODE REVIEW

**Comment by cytec** [comment]
LGTM  \cc @Dr-Bean

File: cross/fdupes/Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/adrianlopezroche/fdupes/archive/
+PKG_DIR = $(PKG_NAME)-$(PKG_NAME)-$(PKG_VERS)
+
+HOMEPAGE = https://github.com/adrianlopezroche/fdupes
+COMMENT  = FDUPES is a program for identifying or deleting duplicate files residing within specified directories.
+LICENSE  = MIT
+
+CONFIGURE_TARGET = none
+GNU_CONFIGURE = 1
```
Comment (Dr-Bean) [inline cross/fdupes/Makefile]: Remove thus, it's unused seeing that you set CONFIGURE_TARGET to none.

File: cross/fdupes/Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+CONFIGURE_ARGS =
```
Comment (Dr-Bean) [inline cross/fdupes/Makefile]: Remove if unused

File: cross/fdupes/Makefile [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+INSTALL_TARGET = myInstall
+
+include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: myInstall
+myInstall:
+	$(RUN) $(MAKE) install PREFIX=$(INSTALL_DIR)/usr/local/fdupes
```
Comment (Dr-Bean) [inline cross/fdupes/Makefile]: It looks better if you use `PREFIX=$(STAGING_INSTALL_PREFIX)`

File: spk/fdupes/Makefile [hunk 4]
```diff
@@ -0,0 +1,23 @@
+SPK_NAME = fdupes
+SPK_VERS = 1.51
+SPK_REV = 1
+SPK_ICON = src/fdupes-logo.png
```
Comment (Dr-Bean) [inline spk/fdupes/Makefile]: For consistency, please call the file `fdupes.png`

File: spk/fdupes/Makefile [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+DEPENDS = cross/fdupes
+
+MAINTAINER = SynoCommunity
```
Comment (Dr-Bean) [inline spk/fdupes/Makefile]: Your name here, as you're the creator and presumably will/should continue to maintain the package.

File: spk/fdupes/src/installer.sh [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Put fdupes in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/bin/fdupes /usr/local/bin/fdupes
+
+    # Set permissions
```
Comment (Dr-Bean) [inline spk/fdupes/src/installer.sh]: This should be unnecessary, please verify.

**Comment by Dr-Bean** [comment]
Couple comments, looks good otherwise.

**Comment by biyiklioglu** [comment]
OK, all done! @Dr-Bean

**Comment by KaraokeStu** [comment]
Published to the repo

**Comment by Tekky001** [comment]
Is this package broken?  I try to start it running DSM 6.1 beta and it goes back to stopped.

Or do I need to use it via CLI?  If so, how?  Thanks!

**Comment by txemi** [comment]
Perhaps newbie question. Above you say "Published to repo". But I do not see fdupes in my DSM package manager. Already community and beta channels activated as I can see them in left menu. 
Any workaround for trying fdupes in my sinology? May I download it and install it from command line?

**Comment by txemi** [comment]
How could I know which one to pick? I found this information in DSM:

DS418
Realtek RTD1296 SoC
DSM 6.2.1-23824 Update 6

Thanks,

**Comment by m4tt075** [comment]
`aarch64-6.1`

**Pull request merged**